### ISSUE TITLE
Advanced search options

### ISSUE THREAD

**Issue by davidamichelson** [srophe/e-gedsh#173]
Let's review the functions here 

What is Title?

Can we please change the Syriaca.org URI label and function to just "URI (e-GEDSH, Syriaca.org, etc.)"

**Comment by davidamichelson** [comment]
We should discuss. Thanks

**Comment by wsalesky** [comment]
@davidamichelson  @dlschwartz  Does this still need to be discussed?

**Comment by dlschwartz** [comment]
I think it does. I was just looking at this yesterday. Maybe we can squeeze this in on Monday when we meet.

**Comment by wsalesky** [comment]
Okay, that would be good. It feels like we are pretty close to finishing up e-gedsh.

**Comment by dlschwartz** [comment]
Yes, I'm trying to push through the final details and get this out asap. Thanks.

**Comment by davidamichelson** [comment]
We discussed this, Winona, please go ahead and change this search box to search any IDNO node and change the label:

Current: Syriaca.org URI 

Revised: URI (e-GEDSH, Syriaca.org, etc.)

**Comment by davidamichelson** [comment]
Let's discuss in person what fields these search in Advanced search.

**Comment by davidamichelson** [comment]
Winona can you please clarify or revise the nodes/paths that these search?

We would expect that searching in:

Keyword: searches everywhere in the text nodes

Place Name: searches in all TEI:placeName text node

Person Name: searches in all TEI:persName text node

Title: Please get rid of this field

Citation: searches in all TEI:bibl and child text nodes

URI (e-GEDSH, Syriaca.org, etc.): searches values of all @ref attributes attached to any element and also all TEI:idno element text nodes?

**Comment by wsalesky** [comment]
@davidamichelson  @dlschwartz Done. Please test and close.

**Comment by dlschwartz** [comment]
@wsalesky We've been careful to encode the TEI with choice, sic, and corr or with choice, reg, orig when we have made changes to the original print edition. In the body, these display just the way we want. However, the content of the sic and orig (not displayed in HTML) show up in search results. See here: http://wwwb.library.vanderbilt.edu/exist/apps/e-gedsh/search.html?q=&placeName=&persName=&bibl=S.+P.+Brock&uri=

**Comment by wsalesky** [comment]
@dlschwartz to make my life easier, can you tell me which of the search results show this problem? They are numbered, so just let me know which number.

**Comment by dlschwartz** [comment]
@wsalesky Sorry, it's the first return. It displays "(see S. P. BrockP. Brock, The Bible in the...". There are many more examples of the same thing elsewhere in the results: 11, 12, 17, 19, etc. 

But now I'm really baffled. Here's what I thought was going on. The print edition uses S. Brock sometimes and S. P. Brock sometimes. I standardized this using choice, reg, and orig to make these all "S. P. Brock". I assumed that this result was showing up because the tei had `<choice><reg>S. P. Brock</reg><orig>S. Brock</reg></choice>.` But I checked the tei for the results I've listed above and that's not the case. The tei actually has `(see S. P. Brock, <hi rend="italic">The Bible in the`. I really don't understand this. Any ideas?

**Comment by wsalesky** [comment]
@dlschwartz this may take some investigating. Let me see what I can find. I will let you know if I have questions about the TEI.

**Comment by wsalesky** [comment]
@dlschwartz  I think this is fixed. Please let me know what you think.

**Comment by dlschwartz** [comment]
@wsalesky Yes, I think this looks good now. In testing search I did find a couple of problems though. 

I did a keyword search for "Peshitta" and got: http://wwwb.library.vanderbilt.edu/exist/apps/e-gedsh/search.html?q=peshitta. 

I did a search for "Yuḥanon" and got: http://wwwb.library.vanderbilt.edu/exist/apps/e-gedsh/search.html?q=Yu%E1%B8%A5anon. Where the previous example seems to be looking for the correct string this error message includes: "err:XPTY0004 xs:string(Dolabani, Philoxenos )"
